tests/general.py

```diff
@@ -97,12 +97,12 @@ def test_detect_hidden_paths(self):
              touch(tmp_fld, "test2.py")]
 
     paths = detect_paths([tmp_fld], hidden=False)
-    self.assertEqual([files[1], files[3]], paths)
+    self.assertItemsEqual([files[1], files[3]], paths)
```

Comment:
Ah, the fix here is to do:
```py
paths.sort()
specific_files = [files[1], files[3]]
specific_files.sort()
```
after line 99 and before line 100.

Reply:
And then keep `self.assertEqual(..)`

Reply:
Yeah `assertItemsEqual` just isn't part of the API in later versions of Pythons unittest. I just wanted to make sure I hadn't broken anything else with my changes to the rules

Reply:
Another solution would be to add `assertItemsEqual()` to `VerminTest` (in "tests/testutils.py")? That would actually be nice.

Reply:
Since the sorting, in both cases, wouldn't be necessary.